bsip-0071.md

```diff
@@ -51,59 +75,126 @@ In my view, a good way to handle bad debt need to follow below principles:
+be filled, not even those with a collateral ratio that would allow them being matched with market orders. 
+This can lead to more bad debt piling up instead of resolving at least some debt positions through margin calls.
+
+Variant B would have let bad debt positions matched at their own collateral ratio while matching other margin calls 
+as before at FP<sub>M</sub> &times; MSSR, hoping to improve the overall collateralization. This also means that debt 
+positions with better collateral ratio would have been preferred over bad debt, meaning that bad debt positions will 
+remain sitting on the market for a longer time.
+
+## The threshhold price
+
+With this BSIP the threshhold price is defined by the asset owner. This may not be considered fair. A suitable 
+future extension is to let asset **holders** have a say in the threshhold price discovery. This could be done by
+introducing a voting procedures where holders of the SmartCoin can participate, with their voting weight reflecting their
+holdings of the SmartCoin in question. This procedure needs precise definition in a new BSIP.
+
+## Voting power
+
+As of right now, a SmartCoin that has BTS as its collateral provides the full amount of collateral as voting power. 
+This is not considered to be fair since it is borrowed voting power and gives incentives to squeeze out as much as 
+possible from a margin position to increase voting weight (borrow, sell bitasset on market, borrow more, and so on). 
+This is adressed by adjusting voting power of a position, and the suggested solution ensures that the voting power of 
+a margin position holders remains unchanged, with the assumption that he sells the newly borrowed bitasset on the 
+respective BTS market.
+
+# Impacts and Risks
+
+ - If the Global Settlement prevention is enabled, the SmartCoin holders lose the ability to instantly settle after a 
+ Global Settlement would have occured, and are forced to use the open market if they want to avoid holding a 
+ SmartCoin that is eventually undercollaterized
+ 
+ - If the Threshhold price is enabled, the promise of being able to settle for equivalent value of the collateral SmartCoin holders are extradited to the asset's owner ability to set it, with the direct implication on the force settlement price. The Threshhold price is defined to prevent external market manipulation, but ultimately the asset owner may use any justification for setting the threshold.
 
-Variant B would have let bad debt positions matched at their own collateral ratio while matching other margin calls as before at FP<sub>M</sub> &times; MSSR, hoping to improve the overall collateralization. This also means that debt positions with better collateral ratio would have been preferred over bad debt, meaning that bad debt positions will remain sitting on the market for a longer time.
+ - If the Threshhold is set, conflict of interest exists if the asset owner is also borrowing the asset into existence
```

Comment:
The asset owner could make an anonymous secondary account so it's safe to assume if the threshold is set there exists a conflict of interest.

Reply:
Yes